modules/update-mirror-repository.adoc

```diff
@@ -89,26 +76,33 @@ $ REMOVABLE_MEDIA_PATH=<path> <1>
 ----
 <1> Specify the full path, including the initial forward slash (/) character.
 
-. Review the images and configuration manifests to mirror:
+. Mirror the version images to the internal container registry:
+** If your mirror host does not have internet access, take the following actions:
+... Connect the removable media to a system that is connected to the internet.
+... Review the images and configuration manifests to mirror:
 +
 [source,terminal]
 ----
+$ oc adm -a ${LOCAL_SECRET_JSON} release mirror \
+     --from=${OCP_RELEASE_REPOSITORY}@${OCP_RELEASE_DIGEST} \
+     --to=${LOCAL_REGISTRY}/${LOCAL_REPOSITORY} \
+     --to-release-image=${OCP_RELEASE_REPOSITORY}@${OCP_RELEASE_DIGEST} --dry-run
 $ oc adm release mirror -a ${LOCAL_SECRET_JSON} --to-dir=${REMOVABLE_MEDIA_PATH}/mirror quay.io/${PRODUCT_REPO}/${RELEASE_NAME}:${OCP_RELEASE}-${ARCHITECTURE} --dry-run
```

Comment:
Mirroring to removable media is useful if you can't mirror directly to the local registry and need to sneakernet the data in.  So when you add the direct-to-local-registry command above while keeping the to-disk command here, it seems like it's conflicted about the network environment.  Maybe these need to get split into restricted-network and connected-network cases, with one command in each case?